.github/workflows/secdb-production.yaml

```diff
@@ -50,4 +50,8 @@ jobs:
 
       - name: 'Upload the security database to a bucket'
         run: |
-          gcloud --quiet alpha storage cp --recursive ${{ github.workspace }}/security.json gs://wolfi-production-registry-destination/os/
+          # Don't cache the security.json.
+          gcloud --quiet alpha storage cp \
+              --cache-control=no-store \
+              ${{ github.workspace }}/security.json \
+              gs://wolfi-production-registry-destination/os/
```

Comment:
I think this change needs to go in the `advisories` repo, since it does the secdb build/publish now

Reply:
How does this look? https://github.com/wolfi-dev/advisories/pull/2